### ISSUE TITLE
read_file order of i3 status not handled

### ISSUE THREAD

**Issue by dohseven** [ultrabug/py3status#1908]
In the [2.13 release](https://github.com/i3/i3status/blob/master/CHANGELOG#L4) of i3status, a new [`read_file`](https://i3wm.org/i3status/manpage.html#_file_contents) order was added. It seems that it's not correctly handled by py3status.

In my i3statusconfig file, I have the following configuration:
```
order += "read_file wallpaper"
read_file wallpaper {
        format = "%content"
        path = "/path/to/wallpaper/description"
}
```

When I run py3status with this configuration, I have the following error:
```
2020-02-25 11:18:41 INFO loading module "read_file wallpaper" from py3status.modules.read_file
2020-02-25 11:18:41 INFO Module `read_file wallpaper` could not be loaded
```

Is it planned to add the support for this order?

**Comment by ultrabug** [comment]
hey @dohseven 

@lasers contributed the fix; I tried it and it works!

I released version 3.27 :)

Thanks!

**Comment by dohseven** [comment]
Hey @ultrabug,
Wow, that was fast! Thank you @lasers for the fix, and thank you @ultrabug for the release, I'll test it as soon as possible!
